datalad/support/annexrepo.py

```diff
@@ -1863,6 +1863,28 @@ def get_description(self, uuid=None):
         else:
             return None
 
+    def set_remote_url(self, name, url, push=False):
+        """Set the URL a remote is pointing to
+
+        Sets the URL of the remote `name`. Requires the remote to already exist.
```

Comment:
since providing custom `__doc__` already anyways (and now just `@borrowdoc`) -- why not to note how this one differs from the parent's? i.e. that it sets annexurl in this case

otherwise LGTM